src/components/SuiteHeader/SuiteHeader.test.jsx

```diff
@@ -72,27 +73,80 @@ describe('SuiteHeader', () => {
     userEvent.click(screen.getByRole('button', { name: 'Close' }));
     expect(screen.getByRole('presentation')).not.toHaveClass('is-visible');
   });
-  it('clicks logout link', () => {
+  it('clicks logout link', async () => {
     delete window.location;
     window.location = { href: '' };
     render(<SuiteHeader {...commonProps} />);
-    userEvent.click(screen.getAllByRole('button', { name: 'Log out' })[0]);
+    await userEvent.click(screen.getAllByRole('button', { name: 'Log out' })[0]);
     expect(window.location.href).toBe(commonProps.routes.logout);
   });
-  it('admin link from admin view takes you to navigator route', () => {
+  it('clicks logout link (but no redirect)', async () => {
+    delete window.location;
```

Comment:
Do we want to store the original value for window.location and restore after test?

Reply:
@davidicus I've added `beforeEach()` and `afterEach()` to all tests that delete `window.location` so that it is restored after each test.